src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
You have way too much faith in Minecraft. Noone would ever bother to put as much work as this would be into it. Especially not if it only concerns modded Minecraft.